src/javascript/PageComposer/PageComposer.jsx

```diff
@@ -134,10 +137,17 @@ export default function () {
         };
     }, []);
 
-    const iFrameOnLoad = () => {
+    const [errorPage, setErrorPageorPage] = useState(false);
```

Comment:
Is there a reason why it is called setErrorPageorPage instead of just setErrorPage ?